app/templates/views/trial-mode.html

```diff
@@ -0,0 +1,45 @@
+{% extends "withoutnav_template.html" %}
+{% from "components/sub-navigation.html" import sub_navigation %}
+
+{% block per_page_title %}
+  Using Notify
+{% endblock %}
+
+{% block maincolumn_content %}
+
+<div class="grid-row">
+  <div class="column-one-third">
+    {{ sub_navigation(navigation_links) }}
+  </div>
+  <div class="column-two-thirds">
+
+    <h1 class="heading-large">Trial mode</h1>
+
+    <p>When you set up a new service it will start in trial mode. This lets you try out GOV.UK Notify, with a few restrictions.</p>
+    <p>A service in trial mode can only:</p>
+      <ul class="list list-bullet">
+       <li>send 50 text messages and emails per day</li>
+        <li>send messages to you and other people in your team</li>
+       <li>create draft letters, but not send them</li>
```

Comment:
Can we avoid introducing a new concept (drafts) here? Maybe something like this instead?
```suggestion
       <li>create letter templates, but not send them</li>
```